src/views/reports/base/BaseReport.vue

```diff
@@ -202,7 +202,7 @@ export default {
     .charts-zone {
 
       ::v-deep {
-        .chart-container{
+        .chart-container {
           break-inside: avoid;
           page-break-inside: avoid;
         }
```

Comment:
The code seems to have no issues or irregularities. However, there is a slight difference in how the width of containers with class "content" was calculated between two places: 

In line 183:

```js
width: 1046px;
```

And in Line 202:

```js
width: 1040px;  
```

Both lines set the same `width`, only they use different prefixes. For consistency and readability, you might choose one prefix either from these options `-1046` (which uses less spaces after minus) or `-1040`. If there's another issue that needs attention, please clarify so I can assist further.